src/pages/account/reset-password.tsx

```diff
@@ -0,0 +1,64 @@
+import { PageHead } from "@/common/components/PageHead";
+import { AuthCard } from "@/common/components/Auth";
+import { Input } from "@/common/components/Input";
+import { useState, useEffect, type MouseEvent } from "react";
+import { supabase } from "@/server/supabase";
+
+export default function ResetPassword() {
+  useEffect(() => {
+    supabase.auth.onAuthStateChange((event, session) => {
+      console.log("event: ", event);
+      console.log("session: ", session);
```

Comment:
I think the logs can be removed

Reply:
removed at https://github.com/AfterClass-io/afterclass.io-v2/pull/22/commits/414605f281116659a90aaf8b024597ad8a49540d !